### ISSUE TITLE
Switch from sSMTP to mhsendmail

### ISSUE THREAD

**Issue by geerlingguy** [geerlingguy/ansible-role-mailhog#3]
Now that the MailHog project also maintains a sendmail stand-in (https://github.com/mailhog/mhsendmail), this role should switch to installing and using _that_ rather than sSMTP, since installation/configuration is a bit simpler.

I'll need to make sure it works correctly with PHP and other layers, and update documentation (and likely some downstream projects that use this role as well)... but this should be a nice way to reduce the weight of this role.

I will likely do a major version bump since this could impact downstream projects by introducing a different bit of software for a major portion of the role's functionality.

**Comment by geerlingguy** [comment]
And, a reference to the original issue where I was tracking this work for MailHog: https://github.com/mailhog/MailHog/issues/45

**Comment by geerlingguy** [comment]
See upstream issue: https://github.com/mailhog/mhsendmail/issues/2

**Comment by geerlingguy** [comment]
Also, inside Drupal, a quick test for this functionality is to update in Drupal VM's `config.yml`:

``` yaml
php_sendmail_path: "/opt/mailhog/mhsendmail"
```

Then use devel to run the following snippet:

``` php
$language = language_default();
$result = drupal_mail('email_example', 'contact_message', 'jeff@jeffgeerling.com', $language, array('message' => 'Test'), 'jeff@jeffgeerling.com', TRUE);
dpm($result);
```

**Comment by geerlingguy** [comment]
And to test that MailHog is accepting emails in general...

```
/opt/mailhog/mhsendmail test@mailhog.local <<EOF
From: App <app@mailhog.local>
To: Test <test@mailhog.local>
Subject: Test message

Some content!
EOF
```

**Comment by geerlingguy** [comment]
New upstream issue: https://github.com/mailhog/mhsendmail/issues/3

Yay for diving into PHP source to work on it a bit more!

**Comment by geerlingguy** [comment]
Upstream issue is resolved, testing locally.

**Comment by geerlingguy** [comment]
Waiting for a new tagged release, 0.2.0

**Comment by pbowyer** [comment]
There is now a new tagged release :) https://github.com/mailhog/mhsendmail/issues/8#issuecomment-196028806

**Comment by geerlingguy** [comment]
I really need to get back to this soon... want to make sure everything works then I'll tag a new release.

**Comment by geerlingguy** [comment]
Taking a look now.

**Comment by geerlingguy** [comment]
Fixed via #4.
